### PR TITLE
Feature/istio

### PR DESCRIPTION
**Pull request by steeling**
add more monitoring including:

* kiali
* jaeger
* more dashboards

### CODE REVIEW

**Comment by steeling** [comment]
renamed to k8s_cluster to get around envoy's use of cluster_name.. and added it to each graph

File: build/deploy/grafana_dashboards/crdb-runtime-grafana.json [hunk 1]
```diff
@@ -81,7 +81,7 @@
       "steppedLine": false,
       "targets": [
         {
-          "expr": "count(up{app=\"cockroachdb\",cluster_name=\"$cluster_name\",pod_name=~\"$node\"})",
+          "expr": "count(up{app=\"cockroachdb\",k8s_cluster=\"$k8s_cluster\",pod_name=~\"$node\"})",
```
Comment (charlie-pisuraj) [inline build/deploy/grafana_dashboards/crdb-runtime-grafana.json:84]: in this case can you change it to k8s_cluster=~\"^$k8s_cluster.*$\"
where ever applicable this way we will get to use multiselect, and aggregate multiple clusters together

File: build/deploy/grafana_dashboards/crdb-runtime-grafana.json [hunk 2]
```diff
@@ -1819,11 +1819,11 @@
         "includeAll": false,
         "label": "Cluster name",
         "multi": false,
```
Comment (charlie-pisuraj) [inline build/deploy/grafana_dashboards/crdb-runtime-grafana.json:1821]: and set this to value to "true" to enable it

File: build/deploy/grafana_dashboards/crdb-runtime-grafana.json (hunk 1 above)
Comment (steeling) [reply build/deploy/grafana_dashboards/crdb-runtime-grafana.json]: done

File: build/deploy/grafana_dashboards/crdb-runtime-grafana.json (hunk 2 above)
Comment (steeling) [reply build/deploy/grafana_dashboards/crdb-runtime-grafana.json]: done

**Pull request merged**